frontend/src/views/upload-project/UploadProjectView.tsx

```diff
@@ -107,7 +107,7 @@ export const UploadProjectView = () => {
   };
 
   if (uploadProjectView === 'success') {
-    return <Redirect to={`/projects/${newProjectId}/edit`} />;
+    return <Redirect to={`/projects/${newProjectId}/edit`} push />;
```

Comment:
There a `<Redirect/>` in `EditProjectNavabar` too, which is not modified here. Not sure if this is expected.